.vscode/settings.template.json

```diff
@@ -4,6 +4,11 @@
     // To use the locally built compiler, after 'npm run build':
     // "typescript.tsdk": "built/local"
 
+    "[typescript][javascript][yaml]": {
```

Comment:
Maybe we can include `json` here too?

Reply:
We have a lot of unformatted JSON, and this PR doesn't attempt to format that or markdown.

Reply:
There's unfortunately some problems when it comes to trailing commas in JSON too, which I need to figure out separately.

Reply:
Fair enough.

Reply:
Though to be clear I absolutely want to get the JSON formatted, but, baby steps 😄